pkg/ovs/openflow/ofctrl_group.go

```diff
@@ -36,13 +36,12 @@ type ofGroup struct {
 // ofSwitch keeps a list of all Group objects, so this operation is
 // needed. Reset() should be called before replaying the Group to OVS.
 func (g *ofGroup) Reset() {
-	// An error ("group already exists") is not possible here since we are
-	// using a new instance of ofSwitch and re-creating a group which was
-	// created successfully before. There will be no duplicate group IDs. If
-	// something is wrong and there is an error, g.ofctrl will be set to nil
-	// and the Agent will crash later.
-	newGroup, _ := g.bridge.ofSwitch.NewGroup(g.ofctrl.ID, g.ofctrl.GroupType)
-	newGroup.Buckets = g.ofctrl.Buckets
+	newGroup := &ofctrl.Group{
```

Comment:
I don't think it is necessary to create a new `*ofctrl.Group`, we could just reset the Switch field in g.ofctrl ?

```
func (g *ofGroup) Reset() {
	g.ofctrl.Switch = g.bridge.ofSwitch
}
```

Reply:
The original code also set `isInstalled` in Group to false. Do we need to set this to false as well?

Reply:
OK.